src/pages/gen1/index.tsx

```diff
@@ -16,7 +16,18 @@ import {
 
 export const meta = {
   title: 'Amplify Docs (Gen 1)',
-  description: 'This is a description for the overview page.'
+  description: 'This is a description for the overview page.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
```

Comment:
Added this so that the directory node created from the meta object knows that this is the right starting point for searching for next and previous links